zerver/webhooks/facebook/doc.md

```diff
@@ -0,0 +1,73 @@
+{!create-stream.md!}
+
+Next, on your {{ settings_html|safe }},
+[create a bot](/help/add-a-bot-or-integration) for
+{{ integration_display_name }}. Make sure that you select
+**Incoming webhook** as the **Bot type**:
+
+![](/static/images/help/bot_types.png)
+
+The API key for an incoming webhook bot cannot be used to read messages out
+of Zulip. Thus, using an incoming webhook bot lowers the security risk of
+exposing the bot's API key to a third-party service.
+
+Construct the URL for the {{ integration_display_name }}
+bot using the bot's API key and the desired stream name:
+
+`{{ api_url }}{{ integration_url }}?api_key=abcdefgh&stream={{ recommended_stream_name }}&token=sampletoken`
+
+Modify the parameters of the URL above, where `api_key` is the API key
+of your Zulip bot, and `stream` is the stream name you want the
+notifications sent to.  
+
+`token` will be used for server verification later, please generate randomly string.
```

Comment:
@nyan-salmon: Okay so I checked Facebook's API docs and this should be replaced with:
```
`token` is an arbitrary string of your choosing that can be used to confirm to your
 server that the request is valid. This string will be included in Facebook's
 incoming payloads each time they send your server a verification request.
```

Reply:
@eeshangarg: 
Sorry...
I have forgot run "git rebase --continue", so My modify was not reflected.

I changed your description from mine.

Reply:
My explanation was as follow:

```
When you enable Webhook in Facebook console, this integration receive a verification request from Facebook.
And this integration will verify `token` matches the value you will enter on Facebook console later.

This is a security check to authenticate the request and identify the Webhook.
So, please generate randomly string.
```